website/content/tutorial-arp.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Tutorial for ARP
+weight: 10
+---
+
+In this tutorial we deploy MetalLB in a (ARM) cluster and announce a load-balanced IP using ARP. We
+assume you have a bare metal [cluster
+running](https://blog.hypriot.com/post/setup-kubernetes-raspberry-pi-cluster/).
+
+The nice thing of (abusing) the ARP protocol is that you don't need any fancy network hardware at
```

Comment:
I don't think we should say abusing here, from the network's POV we are behaving perfectly correctly: we are saying that we own an IP, and when the network sends packets to us, we handle them correctly.